tezos-encoding/Cargo.toml

```diff
@@ -30,7 +30,8 @@ default-features = false
 path = "../tezos-encoding-derive"
 version = "0.5.1"
 
-[features]
-
 [dev-dependencies]
 serde_json = "1.0"
+
+[features]
+blst = ["tezos_crypto_rs/blst"]
```

Comment:
Actually, I don't think you need this? Should be able to update the `crypto_rs` dependency to `default-features = false, features = "std"`.  Data encoding doesn't need any of the signature stuff

Reply:
Actually, we already do that anyway! So can completely remove this `blst` feature :)